.github/workflows/release-dev.yaml

```diff
@@ -0,0 +1,43 @@
+name: Release Dev image
+
+on:
+  push:
+    branches: [main]
+
+jobs:
+  release-dev:
+    name: ${{ matrix.release }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        release: ["release", "release-cross"]
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Set up Go
+        uses: actions/setup-go@v3
+        with:
+          go-version-file: go.mod
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Login to GHCR
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: ${{ matrix.release }}
+        run: make ${{ matrix.release }}
+        env:
+          RELEASE: true
```

Comment:
Why the `RELEASE: true` flag here? Won't it add build assets to the latest github release with each push to main?
Old `deploy.yaml` had `GORELEASER_SKIP_VALIDATE: true` instead